### PR TITLE
GEN-2430 - feat(priceCalculatorv2): added basic recommendation card

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

- Added basic recommendation card. Right now it only links to product page. More to come.

Screenshot 2024-08-29 at 13 38 18

## Justify why they are needed

Price calculator v2 bonus offer feature

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
* **#4667** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4667?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @guilhermespopolin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/BonusOfferPresenter.tsx [hunk 1]
```diff
@@ -1,15 +1,43 @@
...
+  type OfferRecommendationFragment,
+  type ProductRecommendationFragment,
+} from '@/services/graphql/generated'
+import { useShopSessionId } from '@/services/shopSession/ShopSessionContext'
 import { useRoutingLocale } from '@/utils/l10n/useRoutingLocale'
 import { PageLink } from '@/utils/PageLink'
+import { actions } from './BonusOfferPresenter.css'
+import { ProductCardSmall } from './ProductCardSmall'
+
+type Recommendation = {
```
Comment (alebedev) [inline apps/store/src/features/priceCalculator/BonusOfferPresenter.tsx:18]: Can we also use a fragment type here?

**Review by alebedev — APPROVED** [review]
Looks good!

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Aug 29, 8:11 AM EDT**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4667).

**Pull request merged**